libs/metagen/src/tests/fixtures.rs

```diff
@@ -0,0 +1,97 @@
+use crate::interlude::*;
+use common::typegraph::*;
+
+pub async fn test_typegraph_1() -> anyhow::Result<Typegraph> {
+    let out = tokio::process::Command::new("cargo")
+        .args(
+            // "run -p meta-cli -- serialize -f tests/tg.ts"
+            "run -p meta-cli -- serialize -f ../../examples/typegraphs/reduce.py"
+                .split(' ')
+                .collect::<Vec<_>>(),
+        )
+        .kill_on_drop(true)
+        .output()
+        .await?;
+    let mut tg: Vec<Typegraph> = serde_json::from_slice(&out.stdout)
+        .with_context(|| format!("error deserializing typegraph: {out:?}"))?;
+    Ok(tg.pop().unwrap())
+}
+
+#[allow(unused)]
```

Comment:
why unused?

Reply:
Was using this fixture to test cases that are not expressible in the sdk. Will remove once the roundtrip marshalling test suite is done.